src/raghvendra/ArrayMethod.java

```diff
@@ -0,0 +1,64 @@
+package raghvendra;
+
+public class ArrayMethod {
+	int min(int[] number) {
+		int min=number[0];
+		for(int index=0;index<number.length;index++) {
+			if (min>number[index])
+				min=number[index];
+		}
+		return min;
+	}
+	
+	int max(int[] number) {
+		int max=number[0];
+		for(int index=0;index<number.length;index++) {
+			if (max<number[index])
+				max=number[index];
+		}
+		return max;
+	}
+
+	double avg(int[] number) {
+		double average=0;
+		for(int index=0;index<number.length;index++) {
+			average=average+number[index];
+		}
+		return average/number.length;
+	}
+	
+	void display(int[] number) {
+		System.out.println("Elements of number Array: ");
+		for(int index=0;index<number.length;index++) {
+			System.out.print(number[index]+",");
+		}
+	}
+	//Found new line character from google and used it
+	void displayArrayFromEnd(int[] number) {
+		System.out.println( "\r\n" +"Elements of number Array starting from last index: ");
+		for(int index=number.length-1;index>=0;index--) {
+			System.out.print(number[index]+",");
+		}
+	}
+	
+	double avgMinMax(int min, int max) {
+		double average=0;
```

Comment:
I think we have requirement where you need to find max & min number then you need to find avg of both, so this expecting two variable in method looks wrong as per the implementation perspective, user will give us array, not min & max value. I hope you got my point.


You can write code like this, I remove variable and directly doing sum of min & max, no problem, if you are not comfortable to write code like below, just optimize.

int avgOfMinMax = (max(input)+min(input))/2